core/src/test/scala/org/apache/spark/status/AppStatusStoreSuite.scala

```diff
@@ -18,7 +18,10 @@
 package org.apache.spark.status
 
 import org.apache.spark.{SparkConf, SparkFunSuite}
-import org.apache.spark.util.Distribution
+import org.apache.spark.status.LiveEntityHelpers.makeNegative
+import org.apache.spark.status.api.v1
+import org.apache.spark.status.api.v1.{InputMetrics, OutputMetrics, ShuffleReadMetrics, ShuffleWriteMetrics}
```

Comment:
Either use `v1.Foo` everywhere (my preference since other tests of this code use that style), or nowhere.